console/my-account/how-to/send-emails-from-your-instance.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: How to send emails from your Instance
+description: This page shows you how to send emails from your Instance
+dates:
+  validation: 2021-05-26
+  posted: 2021-05-26
+---
+
+<Message type="requirement">
+
+  - You have an account and are logged into the [Scaleway Console](https://console.scaleway.com)
+  - You have created at least one Instance
+
+</Message>
+
+
+By default, remote email ports (25, 465, 587) cannot be reached from our infrastructure to avoid spam. You will need to open the mail ports to send emails. You can enable them from the security group configuration.
+
+
+<Message type="important">
+
+ The SMTP ports are blocked by default for security reasons. To enable them, complete the identification process (KYC) and upgrade your account to Build level.
```

Comment:
I wonder if this should be split between intro & requirements? Eg add to intro:

By default, remote email ports (25, 465, 587) **are blocked**, and cannot be reached from our infrastructure to avoid spam. You will need to open the mail ports to send emails. You can enable them from the security group configuration.

And add to requirements:

- You have [a Build Level account](insert link to relevant how-to)

Reply:
Yes much easier, thank you!